assets/stylesheets/index.css

```diff
@@ -0,0 +1,350 @@
+  padding-bottom: 2rem;
+}
+
+#hero #sub-theme {
+  background: #f8f8f8;
+  color: #515151;
+  font-size: 0.875rem;
+  letter-spacing: -1%;
+  padding: 1.25rem 0.875rem;
+  border: 5px solid #fff;
+}
+
+#hero #date-venue {
+  padding: 0 0.875rem;
+}
+
+#hero #date {
+  color: #595959;
+  letter-spacing: -8%;
+  font-size: 1.5rem;
+  font-weight: 700;
+}
+
+#hero #venue {
+  color: #868686;
+  font-size: 1rem;
+  letter-spacing: -2%;
+}
+
+#program {
+  height: 100%;
+  background-image: url('../../assets/images/program_bg.png');
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  padding: 1rem 0.75rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  color: #fff;
+}
+
+@media screen and (min-width: 768px) {
+  #program {
+    background-image: url('../../assets/images/desktop_program_bg.png');
+    padding: 1rem 10%;
+  }
+
+  #program h1 {
+    margin-bottom: 5px !important;
+  }
+
+  #program article ul {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+    gap: 0.2rem;
+    text-align: center;
+  }
+}
+
+#program > header {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 1rem;
+  padding-bottom: 2.5rem;
+}
+
+#program h1 {
+  font-size: 1.25rem;
+  letter-spacing: -4%;
+  margin-bottom: 1rem;
+}
+
+#program #bar {
+  background: #ff5438;
+  height: 1px;
+  width: 2rem;
+}
+
+.program-card {
+  margin-bottom: 0.4rem;
+  padding: 1rem 0.75rem;
+  display: grid;
+  grid-template-columns: 1fr 1fr 2fr;
+  align-items: center;
+  height: 100%;
+  box-shadow: inset 0 0 0 1000px rgba(59, 62, 69, 0.55);
+}
+
+#program .program-card > img {
+  height: 3.125rem;
+}
+
+#program .program-card > h3 {
+  margin: 0;
+  color: #ff5438;
+  font-size: 0.875rem;
+  line-height: 1.5rem;
+  font-weight: 700;
+  letter-spacing: -1%;
+  padding: 5px;
+}
+
+.program-card > p {
+  margin: 0;
+  font-size: 0.625rem;
+  line-height: 1.5rem;
+  letter-spacing: 6%;
+  font-weight: 700;
+}
+
+#program button {
+  margin: 2rem 1rem;
+  background: #ff4f38;
+  color: #fff;
+  border: none;
+  padding: 1.25rem 3.125rem;
+}
+
+#program #see_program {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  #program article .program-card {
+    display: flex;
+    flex-direction: column;
+  }
+
+  #program #see_program {
+    margin: 3rem 0;
+    display: flex;
+    justify-content: center;
+  }
+
+  #program #see_program a {
+    text-transform: uppercase;
+    color: #fff;
+  }
+}
+
+#speakers {
+  background: #fff;
+  height: 100%;
+  padding: 1.5rem 0.75rem 3rem 0.75rem;
+}
+
+#speakers header {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 1rem;
+  padding-bottom: 2.5rem;
+}
+
+#speakers #speakers-bar {
+  background: #a04038;
+  height: 3px;
+  width: 3rem;
+}
+
+#speakers .speaker-card {
+  display: grid;
+  grid-template-columns: 1fr 3fr;
+  gap: 12px;
+}
+
+#speakers ul .desktop-only {
+  display: none;
+}
```

Comment:
- [ ] It is required to produce a close replica of the guiding template. Your project looks awesome, but you need to look into the margin left  and right of the speaker section compare to what we have on the guiding design. You should also try to look into the space between each speaker picture and the text next it. At the moment, the space on your design is too wide compare to what we have on the guiding template. Kindly compare and adjust your design.

#### Your Design
![your-design-speaker-desktop](https://user-images.githubusercontent.com/26597432/210599599-632db714-cfbd-4135-86f6-ed5c48be7162.png)

### Guiding Template
![temp-speaker-section-desktop](https://user-images.githubusercontent.com/26597432/210599640-b9f711cc-bf24-4d93-8392-4b3dbd5e49eb.png)

Reply:
- [ ] For the mobile version of the same speaker section, there is margin-bottom for each of the speaker for the readability purpose. But there is no such a margin on your design for that particular section which makes defeat the readability feature for that section. Kindly compare your design with the guiding template and work your project along side the guiding template.

### Your Design
![your-speaker-mobile](https://user-images.githubusercontent.com/26597432/210602030-b33610e8-ddfd-497d-80f3-27b939538891.png)

### Guiding template
![temp-speaker-mobile](https://user-images.githubusercontent.com/26597432/210602097-eefab8b7-90f0-4e4e-944f-c05b64806fd3.png)

- [ ] I would also recommend you to add margin left and right for it to look like the original design